soc/arm/nuvoton_npcx/common/soc_clock.h

```diff
@@ -60,6 +60,12 @@ struct npcx_clk_cfg {
 #define APB2DIV_VAL (CONFIG_CLOCK_NPCX_APB2_PRESCALER - 1)
 /* APB3 clock divider, default value (APB3 clock = OSC_CLK/2) */
 #define APB3DIV_VAL (CONFIG_CLOCK_NPCX_APB3_PRESCALER - 1)
+/* APB4 clock divider, default value (APB4 clock = OSC_CLK/6) */
+#ifdef CONFIG_CLOCK_NPCX_APB4_PRESCALER
+#define APB4DIV_VAL (CONFIG_CLOCK_NPCX_APB4_PRESCALER - 1)
+#else
+#define APB4DIV_VAL 0
```

Comment:
Is this to have `inst_cdcg->HFCBCD2 = (APB3DIV_VAL | (APB4DIV_VAL << 4));` working even for chips without APB4? Wondering if it would not be better to have an `#ifdef APB4DIV_VAL` check there and both versions, instead of defining the value to 0, to make it explicit that there is a case that does not set anything there if you read that code and don't know that the define is forced to 0 somewhere else.

Reply:
Got it. Thanks for the suggestion. Will remove APB4DIV_VAL if CONFIG_CLOCK_NPCX_APB4_PRESCALER is not defined later.

Reply:
Done.